app/views/posts/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>Post page</h1>
```

Comment:
- [ ] Kindly make sure for each of the controller actions you have corresponding view files as required for the project requirements. You should add the `app/views/posts/show.html.erb` file in order to have  all the required files for `Post` i.e:
```ruby
     # app/views/users/index.html.erb
     # app/views/users/show.html.erb
```
Right now if you test the `user_path  "users#show"` you will get the below error:
![Screenshot from 2022-11-08 03-16-04](https://user-images.githubusercontent.com/83042742/200450507-ede9ba47-b058-402a-9b23-b348dc61e20f.png)